test/sys/test_mman.rs

```diff
@@ -79,12 +82,17 @@ fn test_mremap_grow() {
 fn test_mremap_shrink() {
     use nix::libc::{c_void, size_t};
     use nix::sys::mman::{mremap, MRemapFlags};
+    use std::num::NonZeroUsize;
 
     const ONE_K: size_t = 1024;
+    let one_k_non_zero = NonZeroUsize::new(ONE_K).unwrap();
+    let ten_one_k = one_k_non_zero
+        .checked_mul(NonZeroUsize::new(10).unwrap())
+        .unwrap();
```

Comment:
I'd hate to raise the MSRV just for this one line in a test.  How about this instead?
```suggestion
    let ten_one_k = NonZeroUsize::new(10 * 1024).unwrap();
```

Reply:
Fixed